rtlib/include/rt/Data.hpp

```diff
@@ -9,9 +9,8 @@
 
 #include <boost/filesystem.hpp>
 
+#include "envi-tools/ENVI.hpp" //IS THIS THE RIGHT WAY TO INCLUDE THIS HEADER?
```

Comment:
Put it in <> braces to indicate it's an external include. Also, you don't need this include here because you don't explicitly use anything from that header in this file. You only need it in `rt/Envi.hpp` because that's where you use a `et::ENVI`